### PR TITLE
Changes agreed to in WG Last Call on "bailiwick"

### PR DESCRIPTION
**Pull request by paulehoffman**

### CODE REVIEW

**Comment by wessels** [comment]
We should be careful not to call name servers glue.  The addresses are the glue. So instead of:
> A modifier to describe a name server whose ...

it should be

> A modifier to describe glue records for a name server whose ...

And similarly instead of:
> A name server's name that is either subordinate to or ...

something like
> Glue records for a name server that is either subordinate to or ...

**Comment by wessels** [comment]
> Glue records for sibling domains are allowed

This doesn't say allowed where.  I think it only makes sense to say "allowed in a referral response"

**Comment by paulehoffman** [comment]
Thanks, @wessels ! Fixed those in the two subsequent commits.

File: draft-ietf-dnsop-rfc8499bis.xml [hunk 1]
```diff
@@ -1290,12 +1290,12 @@ necessarily beyond.</t>
...
 <t>In an NS record, there are three types of relationships between the owner name of the record
 and the name in the NS RDATA: unrelated, in-domain, and sibling domain.
-The application of these three types of relationships is defined in
+The application of these three types of relationships to glue records is defined in
 <xref target="I-D.ietf-dnsop-glue-is-not-optional"/>.
 </t>
 
 <t>An unrelated relationship is one where the NS RDATA contains a name server
-that does not share the same parent as the owner name of NS record
+is not subordinate to the owner name of NS record
```
Comment (wessels) [inline draft-ietf-dnsop-rfc8499bis.xml]: ...that is not subordinate to the zone in which the NS record appears.

Comment (paulehoffman) [reply draft-ietf-dnsop-rfc8499bis.xml]: Looks good to me.

**Comment by kfujiwara** [comment]
Current changes are mostly OK for me.
There are some trivial indications.
1. "In an NS record, there are three types of relationships between the owner name of the record and the name in the NS RDATA"
       "the zone origin" is missing. The part may be "relationships among the owner name of the record, the name in the NS RDATA" and the zone origin.
2. grammarly.com showed some indications
2.1 "An unrelated relationship is one where the NS RDATA contains a name server is not subordinate to the owner name of NS record and therefore is not part of the same zone."
     a name server is -> a name server that is
     of NS record -> of the NS record
2.2 "A sibling domain relationship is one where the NS RDATA contains a name server
whose name that is either subordinate to"
    whose name that is -> whose name is
2.3 "The dictionary definition of bailiwick has been observed to cause more confusion than meaning for this use."
   than meaning -> than the meaning

**Comment by kfujiwara** [comment]
Sorry, I missed pointing one issue.
Current "An unrelated relationship" definition is "NOT in-domain".
Because "a name server that is not subordinate to the owner name of the NS record" may contain "sibling domain".
The text may be "An unrelated relationship is one where the NS RDATA contains a name server that is not subordinate to the zone origin and therefore is not part of the same zone."

**Review by kfujiwara — APPROVED** [review]

**Pull request merged**